cpmpy/expressions/globalconstraints.py

```diff
@@ -285,3 +287,38 @@ def value(self):
     def __repr__(self):
         return "{}[{}]".format(self.args[0], self.args[1])
 
+class XOR(GlobalConstraint):
+    """
+        The 'xor' constraint for more then 2 arguments.
+        Acts like cascaded xor operators with two inputs
+    """
+
+    def __init__(self, arg_list):
+        if len(arg_list) == 2 and is_num(arg_list[1]):
```

Comment:
this is to make it more canonical, e.g. num first? not sure that is really needed, but if you prefer then do add a line-comment perhaps

Reply:
You mean swapping `len(arg_list) == 2` and `is_num(arg_list[1])`?
Might result in anoying errors when length of arg_list is 1...

Reply:
Fixed with extra comment, similar to the one in the Operator constructor